src/sidebars/search/search-controller.js

```diff
@@ -9,6 +9,9 @@
             this.selectionService = selectionService;
             this.currentWindowService = currentWindowService;
 
+            this.quandlService.on('CONNECTION_STATUS_CHAGED', this._setConnectionStatus, this);
```

Comment:
Yeah, my primary reason for suggesting it was because the function's only used once, but I don't mind too much.